### ISSUE TITLE
Building instructions should reference *this* repo

### ISSUE THREAD

**Issue by Gestrid** [ryujinx-mirror/ryujinx#14]
In [step two of the Building instructions](https://github.com/ryujinx-mirror/ryujinx?tab=readme-ov-file#step-2) in the README, the instructions say to use the command `git clone https://github.com/Ryujinx/Ryujinx` when they should say to use the command `git clone https://github.com/ryujinx-mirror/ryujinx`.

**Comment by regginator** [comment]
Indeed, and this includes other links/references to the original repository, `ryujinx.org` domain, CrowdIn, etc.. As of now, I want to keep as much in-tact as possible (besides stuff that could aid in getting this fork potentially taken down) on the upstream `master` branch until more info comes out from the original Ryujinx team.. If all else, I can create a `mirror/master` branch and set that as the default branch for the repo, until we get said additional info.

If we can claim the `Ryujinx` scope on GitHub, I am willing to turn this into a complete hard-fork of the original Ryujinx project, and we *may* be able to attempt to maintain it in some form.

**Comment by Fijxu** [comment]
I don't say this to underestimate but IMO you should not host this on Github nor Gitlab at all. It will be taken down after some days. Same happened with Suyu (Yuzu stripped down basically).

There is always troll people impersonating Nintendo out there. My recommendation is to host it in another git server.

Anyways, here you have more repos of ryujinx mirrored by me: https://git.nadeko.net/Ryujinx_Mirror/ (Ryujinx repo contains the wiki)

**Comment by regginator** [comment]
> @regginator I don't say this to underestimate but IMO you should not host this on Github nor Gitlab at all. It will be taken down after some days. Same happened with Suyu (Yuzu stripped down basically).
> 
> There is always troll people impersonating Nintendo out there. My recommendation is to host it in another git server.
> 
> Anyways, here you have more repos of ryujinx mirrored by me: https://git.nadeko.net/Ryujinx_Mirror/ (Ryujinx repo contains the wiki)
> 
> (Opps, and sorry for going offtopic with the issue! But I wanted to tell you that so feel free to hide this comment later)

It's entirely possible, though do note that the original Ryujinx organization wasn't taken down via a DMCA request, but a private agreement with the maintainer, @gdkchan. Additionally, Ryujinx is licensed under the MIT license, and if doable I can remove any and all external references to `Patreon` and the like.

For now, this repository will primarily serve as a somewhat read-only copy of the original project.

**Comment by regginator** [comment]
Also, fixed in https://github.com/ryujinx-mirror/ryujinx/commit/c6aa83225e52952f03474e5aa5f75044a6fc05b1 :smile:

**Comment by CeruleanSky** [comment]
@Fijxu I am glad I found a copy of [Ryujinx.Graphics.Nvdec.Dependencies](https://git.nadeko.net/Ryujinx_Mirror/Ryujinx.Graphics.Nvdec.Dependencies) , there are a lot of nuget dependencies under the Ryujinx namespace that should probably be cloned and if possible the source code found https://www.nuget.org/profiles/Ryujinx in case they are taken down.
